.babel.react.js

```diff
@@ -43,16 +43,6 @@ module.exports = ({ path }) => ({
                 useBuiltIns: true
             }
         ],
-        [
-            "babel-plugin-named-asset-import",
```

Comment:
I think I already asked about this, but just in case, these are no longer needed?

Reply:
These vere used to preserve loader directives inside imports, like `@svgr/webpack!./local-file.svg`.
There were only few usages of this, and I changed them to be more consistent with everything else.

Because we have `svgr` set on webpack level, we don't need to explicitely define this loader. This makes the configuration a little bit simpler and consistent. Also it allows you to remove all the reduntant `svgo` configuration from all of the projects.

Reply:
Nice.